examples/fuzzer.html

```diff
@@ -0,0 +1,205 @@
+            <div>Maximum Change Interval: <span class="interval-display" id="max-interval-display"></span></div>
+            <paper-slider id="max-interval" pin data-display='#max-interval-display' value="200" min="100" max="5000"></paper-slider>
+            <paper-button raised id="run">Run</paper-button>
+        </div>
+        <model-viewer id="fuzzee"></model-viewer>
+        <script src="./scripts/helpers.js"></script>
+        <script src="./built/dependencies.js"></script>
+        <script src="../dist/model-viewer.js"></script>
+<script>
+const model = $('#fuzzee');
+const updated = $('#update');
+const randomFrom = array => array[Math.floor(Math.random() * array.length)];
+
+const attributes = [{
+    name: 'src',
+    probability: 0.5,
+    values: [
+        'assets/DamagedHelmet/DamagedHelmet.gltf',
+        'assets/BarramundiFish/glTF-Binary/BarramundiFish.glb',
+        'assets/Astronaut.glb',
+        'assets/reflective-sphere.gltf',
+        'i-do-not-exist.gltf',
+    ],
+}, {
+    name: 'background-color',
+    probability: 0.5,
+    values: [
+        '#ff0077',
+        'not-a-color',
+    ],
+}, {
+    name: 'background-image',
+    probability: 0.5,
+    values: [
+        'assets/spruit_sunrise_4k.jpg',
+        'assets/small_hangar_01_1k.jpg',
+        'assets/whipple_creek_regional_park_04_1k.jpg',
+        'i-do-not-exist.jpg',
+    ],
+}, {
+    name: 'controls',
+    probability: 0.2,
+    values: [],
+}, {
+    name: 'auto-rotate',
+    probability: 0.2,
+    values: [],
+}, {
+    name: 'ios-src',
+    probability: 0.5,
+    values: [
+        'assets/Astronaut.usdz',
+        'assets/Astronaut.glb',
+        'i-do-not-exist.usdz',
+    ],
+}, {
+    name: 'magic-leap',
+    probability: 0.2,
+    values: [],
+}, {
+    name: 'preload',
+    probability: 0.2,
+    values: [],
+}, {
+    name: 'poster',
+    probability: 0.2,
+    values: [
+        'assets/poster.png',
+        'assets/poster2.png',
+        'i-do-not-exist.png',
+    ],
+}, {
+    name: 'reveal-when-loaded',
+    probability: 0.2,
+    values: [],
+}, {
+    name: 'unstable-webxr',
+    probability: 0.2,
+    values: [],
+}];
+attributes.forEach(attr => attr.values.push(null, ''));
+
+function updateAttribute (attr, values) {
+    const value = randomFrom(values);
+
+    if (value === null) {
+        model.removeAttribute(attr);
+    } else {
+        model.setAttribute(attr, value);
+    }
+}
+
+let timer = null;
+function tick() {
+    const minInterval = parseInt($('#min-interval').value, 10);
+    const maxInterval = parseInt($('#max-interval').value, 10);
+    for (let attribute of attributes) {
+        if (Math.random() > attribute.probability) {
+            continue;
+        }
+        updateAttribute(attribute.name, attribute.values);
+    }
+
+    updated.style.transition = 'none';
+    updated.style.opacity = 1;
+    requestAnimationFrame(() => {
```

Comment:
This "update" should flash and fade on every change; doesn't work too well for lower values. Could remove

Reply:
Just to indicate to the viewer that a change occurred (more interesting for longer wait times), since sometimes the attribute changes do not result in a visible change

Reply:
Going to remove this for now